app/Http/Controllers/AssignmentController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class AssignmentController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return inertia("AdminPage");
+    }
+
```

Comment:
Just leave this function empty for now rather than render a page that is unrelated.